developer_manual/basics/backgroundjobs.rst

```diff
@@ -85,6 +85,18 @@ or similar things, consider flagging it as time insensitive in the constructor.
 This allows the Nextcloud to delay the job until a given nightly time window so the users
 are not that impacted by the heavy load of the background job.
 
+Configuring parallelism
+^^^^^^^^^^^^^^^^^^^^^^^
+
+With resource-heavy background jobs that run for longer than a few minutes, be they ``QueuedJob`` and ``TimedJob`` instances, you may want to restrict parallelism to prevent multiple such jobs from clogging up your machine's resources. You can do this with the ``setAllowParallelRuns`` method of ``OCP\BackgroundJob\Job`` (``QueuedJob`` and ``TimedJob`` both inherit from this class, so they also have this available).
```

Comment:
```suggestion
With resource-heavy background jobs that run for longer than a few minutes, be they ``QueuedJob`` and ``TimedJob`` instances, you may want to restrict parallelism to prevent multiple such jobs from clogging up the server's resources. You can do this with the ``setAllowParallelRuns`` method of ``OCP\BackgroundJob\Job`` (``QueuedJob`` and ``TimedJob`` both inherit from this class, so they also have this available).
```